ubuntudesign/developer.ubuntu.com
# Add templates and simple homepage

- Added editorconfig and upgraded to django 1.10
- Moved templates to pages subfolder (Happy for better naming suggestions)
- Split up and added @grahambancroft's templates
# QA
- load up server with `./run` command
- check home page and example templates load and look OK
